Source/Array.swift

```diff
@@ -40,6 +40,11 @@ public func >>-<T, U>(a: [T], f: T -> [U]) -> [U] {
     return a.flatMap(f)
 }
 
+@availability(*, unavailable, message="function (T -> U) is not equal to function (T -> [U])")
+public func >>-<T, U>(a: [T], f: T -> U) -> [U] {
+    return a.map(f)
+}
```

Comment:
Is this necessary for arrays? I don't think Swift will coerce `(T) -> U` to `(T) -> [U]` the way it will with optionals.